recipes/libxslt/all/conanfile.py

```diff
@@ -125,9 +125,9 @@ def _build_with_configure(self):
         else:
             configure_args.extend(['--enable-static', '--disable-shared'])
 
-        xml_config = tools.unix_path(self.deps_cpp_info["libxml2"].rootpath) + "/bin/xml2-config"
-
-        configure_args.append('XML_CONFIG=%s' % xml_config)
+        libxml2_path = tools.unix_path(self.deps_cpp_info["libxml2"].rootpath)
+        libxml_src = ("--with-libxml-src=%s" %libxml2_path)
```

Comment:
Just for better readability
```suggestion
        libxml_src = "--with-libxml-src=%s" % libxml2_path
```

Reply:
May simple libxml_src = "--with-libxml-src=" + tools.unix_path(self.deps_cpp_info["libxml2"].rootpath)

What do you think?

Reply:
Done